pkg/api/v1/defaults.go

```diff
@@ -156,6 +156,18 @@ func addDefaultingFuncs() {
 				obj.APIVersion = "v1"
 			}
 		},
+		func(obj *ResourceRequirements) {
+			if obj.Limits != nil {
```

Comment:
Add a note that this sets the requests to the limit if the request was not specified.